python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -718,13 +715,47 @@ def process_graph(args):
                 else:
                     g.edges[etype].data[name] = th.tensor(edata)
         dgl.save_graphs(os.path.join(args.output_dir, args.graph_name + ".dgl"), [g])
+        output_dirs = [args.output_dir]
+    elif args.output_format == "both":
```

Comment:
"both" sounds weird. can we allow users to list the format? for example, "--output-format DGL DistDGL"